package.json

```diff
@@ -40,11 +42,12 @@
     "temp": "0.8.3"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "main": "index.js"
   },
-  "main": "index.js",
+  "main": "lib/jscsrc.json",
```

Comment:
This allows JSCS to find the preset (with the npm package name). ember-cli will use `ember-addon.main` if present so this is a backwards compatible change.